lib/try.js

```diff
@@ -1,15 +1,15 @@
 var express = require('express');
-var bodyParser = require('body-parser');
-var cors = require('cors');
 var request = require('request');
 
 module.exports = function() {
   var proxy = new express.Router();
-  proxy.use(cors());
-  proxy.use(bodyParser());
   
-  proxy.all('/', function(req, res) {  
-    var url = req.query.url;
+  proxy.all('/', function(req, res) {
+    var url = req.headers['x-request-url'];
+    if (!url) {
+      return res.status(500).send('URL must be provided in x-request-url header');
+    }
+    delete req.headers['x-request-url']
```

Comment:
sending out requests as pure as possible (who knows, some endpoints may be especially rigid)